src/main/kotlin/Calculator.kt

```diff
@@ -1,18 +1,20 @@
 import java.lang.NumberFormatException
 
+const val SPLIT_FORMAT = " "
```

Comment:
`Calculator` 외에 사용하는 곳이 없어 보이네요. public이 아닌 private으로 바꾸면 어떨까요?

```suggestion
private const val SPLIT_FORMAT = " "
```